modin/pandas/indexing.py

```diff
@@ -227,6 +227,16 @@ class _LocIndexer(_LocationIndexerBase):
     """A indexer for ray_df.loc[] functionality"""
 
     def __getitem__(self, key):
+        # When getting along a single axis,
+        if not isinstance(key, tuple):
+            # Try to fasttrack the code through already optimized path
+            try:
+                return self.df.__getitem__(key)
+            # This can happen if it is a list of rows
+            except KeyError:
+                pass
+        elif key[0] == slice(None):
+            return self.df.__getitem__(key[1])
```

Comment:
Should we have a try-except block here as well in case it is a list of columns?

Reply:
list of columns should always work here. if it's not a valid argument it will fail in `__getitem__`.

It would be good to add a check for `Series` here, because there should never be a tuple for `Series.loc`. I will add it.